src/kOS/Suffixed/Part/PartValue.cs

```diff
@@ -161,5 +162,15 @@ public override int GetHashCode()
         {
             return !Equals(left, right);
         }
+
+        public static SuffixedList<PartValue> PartsToList(IEnumerable<global::Part> parts, SharedObjects shared)
```

Comment:
you are likely right, i started this wanting to only allow some small subset to be suffixes.

* ISuffixed
* string
* double ( maybe float too)
* int

when sleep deprived chris realized that you cannot specify that kind of constraint in C# he decided that it would shake out in the PR review :)